### PR TITLE
Switch SAs to workload identity

### PR DESCRIPTION
**Pull request by kasiakepka**
**Description**

Changes proposed in this pull request:

- Switch SAs to workload identity: replace presets with service account definition.
[sa-prow-deploy@sap-kyma-prow.iam.gserviceaccount.com](mailto:sa-prow-deploy@sap-kyma-prow.iam.gserviceaccount.com) 
[sa-prow-job-resource-cleaners@sap-kyma-prow.iam.gserviceaccount.com](mailto:sa-prow-job-resource-cleaners@sap-kyma-prow.iam.gserviceaccount.com)
[sa-prowjob-gcp-logging-client@sap-kyma-prow.iam.gserviceaccount.com](mailto:sa-prowjob-gcp-logging-client@sap-kyma-prow.iam.gserviceaccount.com)
[sa-secret-update@sap-kyma-prow.iam.gserviceaccount.com](mailto:sa-secret-update@sap-kyma-prow.iam.gserviceaccount.com) 

- Add sa-kyma-github-issues to external secrets

### CODE REVIEW

File: templates/data/secret-rotation-data.yaml
Comment (dekiel) [inline templates/data/secret-rotation-data.yaml:1]: A service-account parameter is expecting a path to the json key file. Values you provided looks suspicious for me. I think they are not correct locations for this file.

See docs for the tool: https://github.com/kyma-project/test-infra/blob/694650155d5870d925a78f7ebb5b9f81df0995ea/development/gardener-rotate/README.md

**Review by dekiel — CHANGES_REQUESTED** [review]
Please check value provided to the gardener-rotate parameters.

**Comment by kasiakepka** [comment]
@dekiel all changes made to `sa-secret-update` and `preset-sa-secret-update` were reverted.

**Review by dekiel — APPROVED** [review]
Please create a follow up issue to make gardener rotate tool able to work with workload identity setup. The tool should not relay on file with gcp service account credentials. It should be able to work with GOOGLE_APPLICATION_CREDENTIALS variable as far I remember.

**Review by IwonaLanger — APPROVED** [review]

**Pull request merged**